src/config/constants/nftsCollections/types.ts

```diff
@@ -0,0 +1,17 @@
+import { Address } from '../types'
+
+export enum CollectionKey {
+  PANCAKE = 'pancake',
+  SQUAD = 'pancakeSquad',
+}
+
+export type Collection = {
```

Comment:
The Collection constants live to fight another day, for now.

The individual NFTs are all ded.